src/common/utils/secret/helper.ts

```diff
@@ -0,0 +1,16 @@
+import { isNumberString } from 'class-validator';
+
+export const parseValue = (secret_value: string | undefined): (string | number | null) => {
```

Comment:
This function name could be a little more specific. Maybe something like `parseEnvironmentVariable`?

Reply:
Yes, `parseEnvironmentVariable` definitely sounds better.